app/controllers/katello/api/v2/docker_tags_controller.rb

```diff
@@ -25,6 +25,18 @@ def index
       end
     end
 
+    api :GET, "/docker_tags/:id/repositories", N_("List of repositories for a docker meta tag")
+    param :archived, :bool, :desc => N_("include archived repositories")
+    def repositories
+      if ::Foreman::Cast.to_bool params[:archived]
```

Comment:
stylistic choice, up to you here, but we tend to write these things like this:

```
repos = DockerMetaTag.find(params[:id]).repositories
repos = repos.non_archived unless ::Foreman::Cast.to_bool(params[:archived])
```

That said - it looks like you're only using the non-archived version of this API call in the code. maybe we don't need to support querying both ways?

Reply:
I can remove the archived parameter until someone needs it